backend/src/main/java/ca/bc/gov/hlth/hnweb/converter/rapid/RPBSPWB0Converter.java

```diff
@@ -0,0 +1,59 @@
+package ca.bc.gov.hlth.hnweb.converter.rapid;
+
+import org.apache.commons.lang3.StringUtils;
+
+import ca.bc.gov.hlth.hnweb.model.rapid.RPBSHeader;
+import ca.bc.gov.hlth.hnweb.model.rapid.RPBSPWB0;
+import ca.bc.gov.hlth.hnweb.model.rapid.WB0;
+import ca.bc.gov.hlth.hnweb.model.rest.groupmember.AddDependentRequest;
+import ca.bc.gov.hlth.hnweb.model.rest.groupmember.AddDependentResponse;
+
+public class RPBSPWB0Converter extends BaseRapidConverter {
+	
+	private static final String TRAN_CODE = "RPBSPWB0";
+
+	public RPBSPWB0Converter() {
+		super();
+	}
+	
+	public RPBSPWB0 convertRequest(AddDependentRequest request) {
+		RPBSHeader rpbsHeader = new RPBSHeader();
+		rpbsHeader.setOrganization(userInfo.getOrganization());
+		rpbsHeader.setTranCode(getTranCode());
+
+		WB0 wb0 = new WB0();
+		wb0.setGroupNumber(request.getGroupNumber());
+		wb0.setPhn(request.getPhn());
+		wb0.setBenficiaryPHN(request.getDependentPhn());
+		wb0.setCoverageEffectiveDate(formatDate(request.getCoverageEffectiveDate()));
+		wb0.setRelationshipCode(request.getRelationship());
+		wb0.setStudentFlag(request.getIsStudent());
+		if (StringUtils.equals("Y", request.getIsStudent())) {
```

Comment:
Have we used "Y"/"N" for other Boolean flags? Seems like it would be cleaner to use Booleans in the request classes since we have full control over them. Though this avoids the extra translation to fixed width so it's probably ok.

Reply:
I had it as boolean but the extra translations seemed more cumbersome so using a flag as it's only this one instance.